aws/modules/infrastructure_modules/vpc/vpc.tf

```diff
@@ -51,6 +52,7 @@ module "vpc" {
     "isLambda"   = "false"
     "isFirewall" = "false"
     "isDB"       = "true"
+    "Name"       = "${var.vpc_name}-database"
```

Comment:
```tf
private_subnet_suffix  = "private"
database_subnet_suffix = "database"
```

Reply:
Don't these tags already exist because of the above?

Reply:
nope

Reply:
done